include/aspect/material_model/interface.h

```diff
@@ -150,6 +151,13 @@ namespace aspect
       std::vector<double> pressure;
 
       /**
+       * Velocity values at the points given in the #position vector.
+       * This value is mostly important in the case of determining
+       * whether material crossed a certain region (e.g. a phase boundary)
```

Comment:
Period missing. Thinking about this, wouldn't you also need to know about the timestep?